scripts/update-error-codes.sh

```diff
@@ -5,7 +5,7 @@
 set -o errexit -o pipefail
 
 # Constants
-readonly url=https://bit.ly/3i5eThn
```

Comment:
A good improvement here on security! We shouldn't rely on a URL shortener to point us anywhere related to builds.

Reply:
Thanks. I can update the other to scripts that use bitly.